tools/setup_riak

```diff
@@ -2,11 +2,19 @@
 
 sleep 20
 
-RIAK_HOST="http://localhost:8098"
+RIAK_HOST=${RIAK_HOST:-"localhost"}
+RIAK_PORT=${RIAK_PORT:-"8098"}
+RIAK_HOST="http://${RIAK_HOST}:${RIAK_PORT}"
+
+VCARD_SCHEMA_PATH=${VCARD_SCHEMA_PATH:-"tools/vcard_search_schema.xml"}
+MAM_SCHEMA_PATH=${MAM_SCHEMA_PATH:-"tools/mam_search_schema.xml"}
+
+export DOCKER_HOST=${DOCKER_HOST}
```

Comment:
Why do we need assignment here? Seems to be redundant, since this is equivalent to `export DOCKER_HOST`. Also why do we need this export anyway?